### ISSUE TITLE
[Tab Extended]: QA: User is unable to collapse the button using keyboard in 200% zoom.

### ISSUE THREAD

**Issue by gopinathan-n** [carbon-design-system/carbon-for-ibm-dotcom#9237]
### Description

Page Element: Expand Collapse in 200% zoom

Issue Description:
User is unable to collapse the button using keyboard in 200% zoom.

Expected:
User should be able to expand/collapse the button using keyboard.

User Impact:
When the elements are not accessible in 200% zoom, low vision user will face difficulty in accessin the page.

### Component(s) impacted

Tab extended
Tab extended with media

### Browser

Firefox

### Carbon for IBM.com version

v1.36.0-rc.0

### Severity

Severity 3 = The problem is visible or noticeable to users but does not impede the usability or functionality. Affects minor functionality, has a workaround.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

Web component canary: https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components Web component staging: https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
